README.md

```diff
@@ -172,8 +189,110 @@ end
+ end
+end
+
+cyberdyne = { :name => "Cyberdyne", :price => 1897.0, :date => Date.new(2015,1,14) }
+
+stock_writer = StockWriter.new(cyberdyne)
+stock_writer.result
+# => "The price of one share of Cyberdyne on 2015-01-14 is 1897.0 Yen."
+```
+
+`conditional_write` allows Calyx::DataTemplate to choose what grammar rule to invoke. If the condition is true, use the first grammar; otherwise, use the second grammar.
+```ruby
+class GoodStock < Calyx::Grammar
+ start "You should buy stock in <%= name %> because this company has a low EPS."
+end
+
+class BadStock < Calyx::Grammar
+ start "You should sell stock in <%= name %> because this company has a high EPS."
+end
+
+class StockWriter < Calyx::DataTemplate
+ def write_narrative
+  conditional_write eps <= 20, GoodStock, BadStock
+ end
+end
+
+mitsui = { :name => "Mitsui", :eps => 15.8}
+mitsui_writer = StockWriter.new(mitsui)
+mitsui_writer.result
+# => "You should buy stock in Mitsui because this company has a low EPS."
+
+tokoyo_electric = { :name => "Tokyo Electric Power", :eps => 275.2 }
+tokoyo_electric_writer = StockWriter.new(tokoyo_electric)
+tokoyo_electric_writer.result
+# => "You should sell stock in Tokoyo Electric Power because this company has a high EPS."
+```
+
+You may also only provide only one grammar for `conditional_write`. If the condition is false, then nothing will be written.
+```ruby
+class StockWriter < Calyx::DataTemplate
+ def write_narrative
+  conditional_write eps <= 20, GoodStock
+ end
+end
+
+tokoyo_electric = { :name => "Tokyo Electric Power", :eps => 275.2 }
+tokoyo_electric_writer = StockWriter.new(tokoyo_electric)
+tokoyo_electric_writer.result
+# => ""
+```
+
+By simply specifying a few "meta-rules" with conditionals and Grammars, you can generate unique, readable narratives based on your data.
+```ruby
+class StockWriter < Calyx::DataTemplate
+ def write_narrative
+   write StockReport
+   conditional_write eps <= 20, GoodStock, BadStock
+   conditional_write eps <= 10, WonderfulStock
+   conditional_write eps >= 50, AbsolutelyHorribleStock
+   write ThanksForReading
+  end
+end
+```
+
+###
+
 ## License
 
-Calyx is open source and provided under the terms of the MIT license. Copyright (c) 2015 Mark Rickerby
+Calyx is open source and provided under the terms of the MIT license. Copyright (c) 2015 Mark Rickerby, (c) 2016 Tariq Ali
+
+See the `LICENSE` file [included with the project distribution](https://github.com/tra38/calyx/blob/master/LICENSE) for more information.
+
+## History
+In November 2015, Mark Rickerby created Calyx and used that gem to create [choose-your-own adventure gamebooks](https://github.com/dariusk/NaNoGenMo-2015/issues/189). He later on wrote a [blog post](http://maetl.net/notes/storyboard/gamebook-of-dungeon-tropes) explaining his thought process.
+
+In January 2016, Tariq Ali forked Calyx and started adding in new features to turn Calyx into a useful tool for generating data-driven narratives (robojournalism).
+
+##Contributions
```

Comment:
Frankly, this goes a little bit further beyond my initial intentions for this repository.

While I welcome the feedback and am really happy you made something useful out of it, I think that adding contribution guidelines to the README is beyond the scope of what I’d expect to see in a single pull request.

However, I agree contribution guidelines are probably needed. I guess I never really expected to see any contributions to this library, and wasn’t prepared for them, as it was mostly written for my own purposes at the time.